library/src/main/java/io/constructor/data/model/quiz/QuizResultClickRequestBody.kt

```diff
@@ -0,0 +1,30 @@
+package io.constructor.data.model.quiz
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import io.constructor.data.model.common.*;
+import java.io.Serializable
+
+@JsonClass(generateAdapter = true)
+data class QuizResultClickRequestBody(
+    @Json(name = "quiz_id") val quizId: String,
+    @Json(name = "quiz_version_id") val quizVersionId: String,
+    @Json(name = "quiz_session_id") val quizSessionId: String,
+    @Json(name = "result_page") val resultPage: Int?,
+    @Json(name = "result_count") val resultCount: Int?,
+    @Json(name = "num_results_per_page") val numResultsPerPage: Int?,
```

Comment:
Are we missing `result_position_on_page` parameter?

Reply:
👍 Nice catch, looks like this is missing in the public api documentation too. Will add this.